doc/TOR.md

```diff
@@ -0,0 +1,146 @@
+HOWTO USE TOR WITH C-LIGHTNING
```

Comment:
This file is not in markdown format at all. I will be happy to fix it myself. Thanks for the content - I can do the layout.